src/api-reference/financial-integration/v4.financial-integration.markdown

```diff
@@ -429,38 +608,58 @@ Code|Description|Category
 
 Name|Type|Format|Description
 ---|---|---|---
-`fIDocuments`|`Array`|[Financial document](#schema-fidocument)|The result collection.
-`nextPage`|`string`|-|The URI of the next page of results.
-`numberReturned`|`int32`|-|Number of documents returned in current page.
-`offset`|`string`|-|Next page offset.
+`content`|`Array`|[FIDocument](#schema-FIDocument)|The result collection.
+`page`|`string`|[PageMetadata](#schema-PageMetadata)|Pagination details
+`links`|`Array`|[Link](#schema-Link)|Pagination links.
 
-### <a name="schema-fidocument"></a>Financial Document
+### <a name="schema-FIDocument"></a>FIDocument
 
 Name|Type|Format|Description
 ---|---|---|---
 `id`|`string`|`-`|The unique identifier for the document.
-`docType`|`string`|-|Transaction type. Supported values are: `expense`, `invoice`, or `cashadvance`.
+`docType`|`string`|-|Transaction type. Supported values are: `expense`, `invoice`, `cashadvance`, `payroll`, `obligation`.
 `companyId`|`string`|-|Unique identifier for the company in Concur.
 `entityId`|`string`|-|Unique Identifier for the entity in Concur.
 `companyUuid`|`string`|`UUID`|Unique UUID for the company in Concur.
 `erpSystemId`|`string`|-|The external System ID that processed the document.
 `document`|`Array`|-|The JSON financial document. Review the [FI sample documents](#examples) below
 `docStatus`|`string`|-|The financial document status. Supported values are: `READY`, `ACKNOWLEDGED`, `POSTING_CONFIRMED_SUCCESS`, `POSTING_CONFIRMED_FAILURE`
 
+### <a name="schema-PageMetadata"></a>PageMetadata
+
+Name|Type|Format|Description
+---|---|---|---
+`size`|int32|`-`|Total number of pages returned
+`totalElements`|int32|-|Total count of records returned
+`totalPages`|int32|-|Total number of pages returned
+`number`|int32|-|Page location for which the result is returned, ex: first page starts with number 0, second page starts with 1.
+
+
+### <a name="schema-Link"></a>Link
+
+Name|Type|Format|Description
+---|---|---|---
+`rel`|`string`|`-`|Relation of link, ex: `first`, `self`, `next`, `last`
+`href`|`string`|-|Complete URL for the paginated link
+`media`|`string`|-|null
```

Comment:
These descriptions can't be null, they need descriptive text.

Reply:
done